src/main/java/org/prebid/server/bidder/facebook/FacebookBidder.java

```diff
@@ -207,14 +208,32 @@ private static Banner modifyBanner(Imp imp, boolean impInstlEqOne) {
             return banner.toBuilder().w(0).h(0).format(null).build();
         }
 
-        if (!Objects.equals(banner.getH(), 50) && !Objects.equals(banner.getH(), 250)) {
+        if (banner.getH() == null) {
+            for (int i = 0; i < banner.getFormat().size(); i++) {
```

Comment:
1. Can `banner.getFormat()` be null?
2. Can we use `foreach` here for netter readability?